src/api/taskRuns.js

```diff
@@ -78,6 +78,7 @@ export function cancelTaskRun({ name, namespace }) {
 export function createTaskRun({
   namespace,
   taskName,
+  taskRunName,
```

Comment:
same as for PipelineRun, we should have the default value in the API:
```diff
-  taskRunName,
+  name = `${taskName}-run-${Date.now()}`,
```

Reply:
Thanks for feedback :D reflected in https://github.com/tektoncd/dashboard/pull/2345/commits/b68845a1584c1128e3e31992e1e764d541615d34